lib/neo4j/http/node_client.rb

```diff
@@ -26,7 +28,9 @@ def upsert_node(node)
         results.first&.fetch("node")
       end
 
-      def delete_node(node)
+      def delete_node(node, unwind: nil)
+        return delete_node_via_unwind(node, unwind) if unwind.present?
```

Comment:
^^ This makes me thing - 100% we should, yes.  I don't see the value in overriding `delete_mode`, but maybe I'm missing it